Exiled.API/Features/Attributes/Validators/GreaterThanAttribute.cs

```diff
@@ -12,29 +12,30 @@ namespace Exiled.API.Features.Attributes.Validators
     using Exiled.API.Interfaces;
 
     /// <summary>
-    /// An attribute to easily manage config values.
+    /// An attribute that validates if the value of the marked property is greater than a specified number.
     /// </summary>
     [AttributeUsage(AttributeTargets.Property)]
     public sealed class GreaterThanAttribute : Attribute, IValidator
     {
         /// <summary>
         /// Initializes a new instance of the <see cref="GreaterThanAttribute"/> class.
         /// </summary>
-        /// <param name="number">A number the value should be greater.</param>
-        /// <param name="isIncluded">Whether or not <paramref name="number"></paramref> is included.</param>
-        public GreaterThanAttribute(object number, bool isIncluded = false)
+        /// <param name="number">The number the marked property should be greater than.</param>
+        /// <param name="isIncluded">Whether or not the comparison in inclusive (includes <see cref="Number"/> as a valid value for the marked property).</param>
+        public GreaterThanAttribute(IComparable number, bool isIncluded = false)
```

Comment:
Don't change it, attributes unfortunately don't support it, that's why I've done it like this

Reply:
Oh, interesting. Sorry